mix.exs

```diff
@@ -47,8 +47,10 @@ defmodule EpiViewpoint.MixProject do
       local_or_remote(:remote, :euclid, version: "~> 0.1", path: System.get_env("EUCLID_PATH", "../euclid")),
       {:bcrypt_elixir, "~> 2.1"},
       {:cowboy_telemetry, "~> 0.4"},
+      {:dart_sass, "~> 0.7", runtime: Mix.env() == :dev},
       {:ecto_sql, "~> 3.11"},
       {:eqrcode, "~> 0.1"},
+      {:esbuild, "~> 0.7", runtime: Mix.env() == :dev},
```

Comment:
While `~> 0.7` will match `0.8` I'd prefer if we match on the latest minor version. Same for `:dart_sass` too.

Reply:
the latest `dart_sass` is 0.7.1, will update `esbuild` to 0.8